src/constructor.hpp

```diff
@@ -246,7 +260,7 @@ class Constructor : public Progressive {
      * one FASTA file. Reference and VCF vectors may not contain nulls.
      */
     void construct_graph(const vector<FastaReference*>& references, const vector<vcflib::VariantCallFile*>& variant_files,
-        function<void(Graph&)> callback);
+        function<void(Graph&)> callback, bool do_svs, const vector<FastaReference*>& insertions);
```

Comment:
I think do_svs maybe should be a configuration flag on Constructor.

Why are the insertion FASTAs separate from the reference FASTAs?

Also, the callback probably should stay as the last parameter, so you don't have a long inline lambda and then more arguments to the function it gets passed to after it.